frontend/src/store/slices/menu/middleware.ts

```diff
@@ -0,0 +1,15 @@
+import { AppDispatch } from "store/store";
+
+import slice from "./slice";
+
+const { setMenuActiveItem, setMenuOpenDrawer } = slice.actions;
+
+const activateMenuItem = (value: string[]) => (dispatch: AppDispatch) => {
+  dispatch(setMenuActiveItem(value));
+};
+
+const openMenuDrawer = (value: boolean) => (dispatch: AppDispatch) => {
+  dispatch(setMenuOpenDrawer(value));
+};
+
+export default { activateMenuItem, openMenuDrawer };
```

Comment:
`Never have two values in object in the same line` huh?
instead of `export default { a, b }` you can use `export const a =....` and `export const b = ...` (I did not checked the code yet)